spec/user_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+require 'user'
+
+RSpec.describe User do
+	#your test code here
```

Comment:
Incorrect indentation detected (column 1 instead of 2).<br>Missing space after #.<br>Tab detected.